app/models/credit_note.rb

```diff
@@ -15,13 +15,14 @@ class CreditNote < ApplicationRecord
 
   has_one_attached :file
 
-  monetize :amount_cents
-  monetize :remaining_amount_cents
+  monetize :total_amount_cents
+  monetize :credit_amount_cents
+  monetize :balance_amount_cents
```

Comment:
can those fields be nil in certain scenarios? I remember I had bug around nil value and `monetize`...